src/left-nav.jsx

```diff
@@ -82,7 +82,10 @@ const LeftNav = React.createClass({
   //from the parent / owner using context
   componentWillReceiveProps (nextProps, nextContext) {
     let newMuiTheme = nextContext.muiTheme ? nextContext.muiTheme : this.state.muiTheme;
-    this.setState({muiTheme: newMuiTheme});
+    this.setState({
+      muiTheme: newMuiTheme,
+      open: this.props.docked && !nextProps.docked ? false : this.state.open || nextProps.docked,
```

Comment:
Can't this be `open: (this.props.docked !== nextProps.docked) ? nextProps.docked : this.state.open `?